packages/orchestration/README.md

```diff
@@ -17,16 +17,145 @@ $ npm install @sap-ai-sdk/orchestration
+
+You can pass configuration information like for example the LLM to be used for inference and template, as a parameter to the client.
+
+#### Orchestration client with Templating
+
+Use the Orchestration client with templating to pass a prompt with placeholders, that can be filled during a chat completion request with the input parameters.
+
+This feature enables you to vary the prompt based on the input parameters.
+
+```TS
+import { OrchestrationClient } from '@sap-ai-sdk/orchestration';
+
+const orchestrationClient = new OrchestrationClient({
+  llm: {
+    model_name: 'gpt-4-32k',
+    model_params: { max_tokens: 50, temperature: 0.1 }
+  },
+  templating: {
+    template: [
+      { role: 'user', content: 'What is the capital of {{?country}}?' }
+    ]
+  }
+});
+
+const response = await orchestrationClient.chatCompletion({
+  inputParams: { country: 'France' }
+});
+
+const responseContent = response.getContent();
+```
+
+It is possible to provide a history of a conversation to the model.
+Use the following snippet to send a chat completion request with history and a system message:
+
+```TS
+import { OrchestrationClient } from '@sap-ai-sdk/orchestration';
+
+const orchestrationClient = new OrchestrationClient({
+  llm: {
+    model_name: 'gpt-4-32k',
+    model_params: { max_tokens: 50, temperature: 0.1 }
+  },
+  templating: {
+    template: [
+      { role: 'user', content: 'What is my name?' }
+    ],
+    messages_history: [
+      {
+        role: 'system',
+        content: 'You are a helpful assistant who remembers all details the user shares with you.'
+      },
+      {
+        role: 'user',
+        content: 'Hi! I am Bob'
+      },
+      {
+        role: 'assistant',
+        content: 'Hi Bob, nice to meet you! I am an AI assistant. I will remember that your name is Bob as we continue our conversation.'
+      }
+    ]
+  }
+});
+
+const responseContent = await orchestrationClient.chatCompletion().getContent();
+const tokenUsage = response.getTokenUsage();
+
+logger.info(
+  `Total tokens consumed by the request: ${tokenUsage.total_tokens}\n` +
+  `Input prompt tokens consumed: ${tokenUsage.prompt_tokens}\n` +
+  `Output text completion tokens consumed: ${tokenUsage.completion_tokens}\n`
+);
+```
+
+#### Orchestration client with Content Filtering
 
-This project is open to feature requests/suggestions, bug reports etc. via [GitHub issues](https://github.com/SAP/ai-sdk-js/issues).
+Use the Orchestration client with filtering to restrict the type of content that is passed to and received from a generative AI model.
+
+This feature enables you always filter both input and the output of a model based on content safety criteria.
+
+```TS
+import { OrchestrationClient } from '@sap-ai-sdk/orchestration';
+
+const filter = buildAzureContentFilter({ Hate: 2, Violence: 4 });
```

Comment:
[pp] Consider commenting on the fact that this is a convenience function.

Reply:
[req] the function should also be in the import statement